api-catalog-ui/frontend/src/components/Wizard/WizardInputs.jsx

```diff
@@ -61,7 +112,12 @@ class WizardInputs extends Component {
     };
 
     render() {
-        return <div className="wizardForm"> {this.loadInputs()}</div>;
+        return (
+            <div className="wizardForm">
+                {this.loadInputs()}
+                {this.props.data.multiple ? <Button onClick={this.addFields}>Add more fields</Button> : null}
+            </div>
```

Comment:
What about adding a button to remove the additional fields? (like a `-` icon). Just was thinking that once the validation will be in place, if the user adds them by mistake, he then cannot remove them.

Reply:
That makes sense - we will add it and push it to the pr

Reply:
Thank you :)!